queries/matlab/indents.scm

```diff
@@ -0,0 +1,32 @@
+((keyword) @indent.end @indent.branch
+  (#eq? @indent.end "end")
+  (#eq? @indent.branch "end"))
+
+[(if_statement)
+ (for_statement)
+ (while_statement)
+ (switch_statement)
+ (try_statement)
+ (function_definition)
+ (class_definition)
+ (enumeration)
+ (events)
+ (methods)
+ (properties)
+ ] @indent.begin
+
+[(elseif_statement)
+ (else_statement)
+ (case)
+ (otherwise)
+ (catch)
+ ] @indent.branch
```

Comment:
It's the entire block. I tested it as I did it, and it works. They look like

```
(else
 else: (keyword)
(block))
```

Reply:
I'd say to only have the `keyword` of `else: (keyword)` be `@indent.branch` and similarly to the other in these sections, the whole block doesn't really need it. It will look at the smallest node at the start of the line to consider for self-dedenting, and capturing such node will be better

Reply:
Thank you for clarifying how it works. I changed accordingly. Does the same hapen with `@indent.begin`? All the grammars I saw put that one in the whole tag...

Reply:
Generally, yes. But no need to change `@indent.begin`,as your current version aligns with other `@indent.begin` in other `indents.scm` file